caps/static/js/main.js

```diff
@@ -121,4 +121,25 @@ $(function() {
         );
     });
 
+    $("a").click(function(e) {
+        link = $(this);
+        if (link.data("plan-id")) {
+            link_url = link.attr('href');
+            link_text = link.text();
+            link_ext = "." + link_url.split('.').pop();
+            file_name = link_url.split('/').pop();
+            council_slug = link.data("council-slug");
+            plan_id = link.data("plan-id");
+            gtag('event', 'plan_link_click', {
```

Comment:
I _think_ there’ll be a race condition here, where the communication with Google might not have a chance to happen before the default action of the link kicks in and the page reloads.

The gtag docs include an example of passing an `event_callback` parameter along with the other event parameters, to specify code that should run _after_ the event has been successfully logged:

https://developers.google.com/analytics/devguides/collection/gtagjs/sending-data#implement_event_callback_functions

Back in the universal analytics days, we had a helper library for doing all of this: https://github.com/mysociety/analytics#analyticstracklinkclicke-params – maybe you could update the helper library to use `gtag()` if it’s available, and then we could use that?